DigitalLearningSolutions.Data.Tests/Services/GroupServiceTests/GroupsServiceTests.cs

```diff
@@ -662,6 +663,82 @@ public void
             }
         }
 
+        [Test]
+        public void AddNewDelegateToRegistrationFieldGroupsAndEnrolOnCourses_adds_delegate_to_groups()
+        {
+            // given
+            var now = DateTime.Now;

```

Comment:
Set this to some fixed date in the future (e.g. `new DateTime(3000, 1, 1)`)

https://leanylabs.com/blog/good-unit-tests/#write-deterministic-tests

https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/20067450883/Use+ClockService+instead+of+DateTime.UtcNow+or+DateTime.Today